### PR TITLE
All documentations for the final sprint.

### PR DESCRIPTION
**Pull request by ClintonCao**
I've added all the final documents for the final sprints. Please review them and let me know if there are any questions/suggestions.

### CODE REVIEW

**Comment by boninggong** [comment]
Wow so many documents, looking good!

**Comment by boninggong** [comment]
Sorry wrong button

**Comment by ClintonCao** [comment]
It's okay ;)

File: README.md [hunk 1]
```diff
@@ -60,10 +60,11 @@ Maven Execution Research [maven execution research](https://github.com/ClintonCa
 
 Testing Documentation [testing documentation](https://github.com/ClintonCao/Contextproject-TSE/blob/master/documentation/Testing%20related%20documents/TestingDocumentation.md)
 
-## Link to SIG Feedback 1 (Picture):
-SIG Feedback: [SIG Feedback](https://github.com/ClintonCao/Contextproject-TSE/blob/master/documentation/SIG%20Feedback/SIGFeedback.JPG)
+## Link to SIG Feedback & Code Improvements:
+The feedback of SIG and the code improvements can be found [here](https://github.com/ClintonCao/Contextproject-TSE/tree/master/documentation/SIG%20Feedback).
```
Comment (SunweiWang) [inline README.md]: I think the link here should be following:
https://github.com/ClintonCao/Contextproject-TSE/blob/master/documentation/SIG%20Feedback/SIGFeedback.JPG

Comment (ClintonCao) [inline README.md]: @SunweiWang No it is right, read the title and what is actually in the folder.

Comment (ClintonCao) [inline README.md]: It does not take you to the picture anymore, but the whole folder containing the feedback.

Comment (SunweiWang) [inline README.md]: :ok_hand:

**Comment by Michieldoesburg** [comment]
Looks good to me. I think we can merge, anyone else?

**Pull request merged**